DiscordChatExporter.Cli.dockerfile

```diff
@@ -1,22 +1,33 @@
 # Build
-FROM mcr.microsoft.com/dotnet/sdk:7.0 AS build
+FROM mcr.microsoft.com/dotnet/sdk:7.0-alpine AS build
+
+WORKDIR /build
 
 COPY favicon.ico ./
 COPY NuGet.config ./
 COPY Directory.Build.props ./
 COPY DiscordChatExporter.Core ./DiscordChatExporter.Core
 COPY DiscordChatExporter.Cli ./DiscordChatExporter.Cli
 
-RUN dotnet publish DiscordChatExporter.Cli --configuration Release --output ./publish
+RUN dotnet publish DiscordChatExporter.Cli \
+    --self-contained \
+    --use-current-runtime \
+    --configuration Release \
+    --output ./publish
 
 # Run
-FROM mcr.microsoft.com/dotnet/runtime:7.0 AS run
+FROM mcr.microsoft.com/dotnet/runtime-deps:7.0-alpine
 
-RUN useradd dce
+RUN apk add --no-cache tzdata && \
+    mkdir -p /opt/discord_chat_exporter && \
+    adduser \
+    --disabled-password \
+    --no-create-home \
+    dce
 USER dce
+COPY --from=build /build/publish /opt/discord_chat_exporter
 
-COPY --from=build ./publish ./
-
-WORKDIR ./out
+WORKDIR /out
 
-ENTRYPOINT ["dotnet", "../DiscordChatExporter.Cli.dll"]
\ No newline at end of file
+ENV PATH="$PATH:/opt/discord_chat_exporter"
+ENTRYPOINT ["DiscordChatExporter.Cli"]
```

Comment:
Can we just do `ENTRYPOINT ["/opt/discord_chat_exporter/DiscordChatExporter.Cli"]` here by the way? Or is there some extra benefit from having it on the path?

Reply:
Yeah, you can use the full path. Having it in `$PATH` allows you to easily launch the app if you need to debug something while running an interactive shell session with `docker run --entrypoint /bin/sh -it --rm tyrrrz/discordchatexporter:stable`.

Reply:
I see, makes sense. Can you add a comment to explain that so I don't forget? :)

Reply:
Done!